examples/no-code/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "no-code",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "serve": "vite preview"
+  },
+  "dependencies": {
+    "ra-data-local-storage": "^3.14.0",
+    "ra-no-code": "^0.0.1",
+    "react": "^17.0.0",
+    "react-admin": "^3.14.4",
```

Comment:
It uses react-admin components

Reply:
But your example only imports from 'ra-core'. It doesn't feel natural to force users to add a package that they don't have to use. 


I think it stems from the fact that 'react-admin' is a peerDependency of 'ra-no-code'.

Reply:
It is because we actually use components from the `react-admin` package in addition from `ra-core` and `ra-ui-materialui`. You want it to be a dependency instead?

Reply:
Yes, I think users shouldn't have to add a dependency that they don't actively use. If you need react-admin to use ra-no-code, then ract-admin should be a dependency of ra-no-code, not a peerDependency